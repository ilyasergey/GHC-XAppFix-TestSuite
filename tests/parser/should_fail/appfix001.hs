-- Parsing the local 'alet' form

{-# LANGUAGE ApplicativeFix #-}

module ShouldCompile where

foo = alet a = undefined
           Just(b) = undefined
        in a
