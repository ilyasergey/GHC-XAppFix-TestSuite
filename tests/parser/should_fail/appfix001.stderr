
appfix001.hs:8:12:
    No pattern bindings allowed in alet: Just (b) = undefined
